src/main/java/uk/ac/cam/cl/dtg/isaac/dto/GameboardDTO.java

```diff
@@ -30,7 +30,7 @@
 public class GameboardDTO {
     private String id;
     private String title;
-    private List<GameboardItem> questions;
+    private List<? extends GameboardItem> questions;
```

Comment:
Is this syntax necessary? I though subclasses could be included in lists of the parent class?

Reply:
https://stackoverflow.com/a/3009779